pkg/options/options.go

```diff
@@ -136,8 +136,8 @@ var ShowSecrets = false
 var ManualDefaultBackingStore = false
 
 // DefaultAutoscalerType is the default noobaa-endpoint autoscaler type
-// it can be overridden for testing or different types. for now default is hpav1 it will replace with hpav2 in future
-var DefaultAutoscalerType = "hpav1"
+// it can be overridden for testing or different types. there is no default autoscaler for endpoint
+var DefaultAutoscalerType = ""
```

Comment:
what is an empty default? 
Is that mean that in default no autoscaling will happen?

Reply:
`Are the changes in the go mod file correspond with ODF?` -> As @liranmauda suggested I am checking this Boris by creating an ODF private build with `naveenpaul1:hpav2-autoscale` branch. He is busy right now once Boris done the build will update here.